examples/rocket_example/api/src/lib.rs

```diff
@@ -145,19 +156,107 @@ async fn run_migrations(rocket: Rocket<Build>) -> fairing::Result {
 
 #[tokio::main]
 async fn start() -> Result<(), rocket::Error> {
-    rocket::build()
+    let mut building_rocket = rocket::build()
         .attach(Db::init())
         .attach(AdHoc::try_on_ignite("Migrations", run_migrations))
         .mount("/", FileServer::from(relative!("/static")))
         .mount(
             "/",
             routes![new, create, delete, destroy, list, edit, update],
         )
+        .mount(
```

Comment:
Actually, I think it might make sense we make a new example instead of piggy-backing the original example? may be `rocket_okapi_example`?

Reply:
Sure @tyt2y3 I will fix all observations. Glad to help! :)